src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -904,12 +903,15 @@ export default class Aircraft {
 
         let isExpeditingString = '';
         if (expedite) {
-            isExpeditingString = 'and expedite';
+            isExpeditingString = ' and expedite';
         }
 
+        const newAltitude = this.fms.altitudeForCurrentWaypoint();
+        const radioTrendAltitude = radio_trend('altitude', this.altitude, newAltitude);
+        const currentWaypointRadioAltitude = radio_altitude(newAltitude);
         const readback = {
-            log: `${radioTrendAltitude} ${this.fms.altitudeForCurrentWaypoint()} ${isExpeditingString}`,
-            say: `${radioTrendAltitude} ${currentWaypointRadioAltitude} ${isExpeditingString}`
+            log: `${radioTrendAltitude} ${newAltitude}${isExpeditingString}`,
+            say: `${radioTrendAltitude} ${currentWaypointRadioAltitude}${isExpeditingString}`
```

Comment:
I would prefer that `isExpeditingString` not have the space and that any spacing would happen here within the template string itself.  Is there a reason you added it back in to `isExpeditingString` variable?